src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const threeDays = 3;
+  const sevenDays = 7;
```

Comment:
not the best names for days
what if we will change number? You will need to rename variable
better smth like `daysForSmallDiscount`